draft-irtf-cfrg-hpke.md

```diff
@@ -273,50 +275,90 @@ values are two bytes long.
 
 ## DH-Based KEM
 
-Suppose we are given a Diffie-Hellman group that provides the
-following operations:
+Suppose we are given an instantiation KDF_kem of KDF with functions
+Hash_kem, Extract_kem, and Expand_kem, and a Diffie-Hellman group
+providing the following operations:
 
 - GenerateKeyPair(): Generate an ephemeral key pair `(sk, pk)`
   for the DH group in use
 - DH(sk, pk): Perform a non-interactive DH exchange using the
-  private key sk and public key pk to produce a shared secret
-  of length Npk
-- Marshal(pk): Produce a fixed-length byte string
+  private key sk and public key pk to produce a Diffie-Hellman
+  shared secret of length Ndh
```

Comment:
Can we define `Ndh` on its own line? (I missed it the first time around.) Is it even needed?

Reply:
This was a suggestion by @bifurcation, with the background that there might be a DH scheme where the result of the DH operation is not an encoded public key.

We can move it to it's own line I think.